newlib/libc/tinystdio/vfprintf.c

```diff
@@ -1137,8 +1172,12 @@ int vfprintf (FILE * stream, const CHAR *fmt, va_list ap_orig)
 #endif
                 }
 #endif
-#ifdef _PRINTF_PERCENT_N
```

Comment:
Let's not change the semantic of how vfprintf handles %n; I think it's best if it appears in the output instead of just disappearing.